packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.scss

```diff
@@ -0,0 +1,62 @@
+@use 'usa-step-indicator/src/styles/usa-step-indicator';
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@import '../../mixins/accessibility.css';
+
+@import '../../global/formation_overrides';
+
+
+
+.usa-step-indicator__segment::after {
+  background-color: #919191; // USWDS design and sketch
+}
+@media (min-width: 40em) {
+  .usa-step-indicator--counters .usa-step-indicator__segment::before, .usa-step-indicator--counters-sm .usa-step-indicator__segment::before {
+    box-sizing: border-box;
+    box-shadow: inset 0 0 0 rem-override(0.25rem) #919191, 0 0 0 rem-override(0.25rem) #fff // USWDS design and sketch
+  }
+
+  .usa-step-indicator--counters .usa-step-indicator__segment--current::before, .usa-step-indicator--counters-sm .usa-step-indicator__segment--current::before,
+  .usa-step-indicator--counters .usa-step-indicator__segment--complete::before, .usa-step-indicator--counters-sm .usa-step-indicator__segment--complete::before {
+    box-shadow: 0 0 0 rem-override(0.25rem) #fff;
+  }
+}
+
+.usa-step-indicator__segment--current::after {
+  background-color: #005EA2; // USWDS design and sketch
+}
+
+.usa-step-indicator__segment--complete::after {
+  background-color: #162E51; // USWDS design and sketch
+}
```

Comment:
For all of the USWDS style overrides above, are these all VA specific decisions? If not then why aren't they the same as what the USWDS module is providing for those classes?

Reply:
I was a little confused on this as well. The color provided by the module for default/incomplete was different than what both the uswds site and our sketch designs specify. I didn't know if I should override it or not, but chose to do so and hoped to get feedback on what should happen (I had to add these colors (lines 25 and 29) because specifying the correct default/incomplete color on line 10 overwrote the current and completed colors). @LillyBoot the incomplete/default color from the module is #dfe1e2, all the designs say it should be #919191. What should be done here?

Module styles:
![Screenshot 2023-05-01 at 2 10 57 PM](https://user-images.githubusercontent.com/1776069/235522624-c0ffd45a-1291-4e29-9e98-87b7b25369ee.png)


Overwritten to match design:
![Screenshot 2023-05-01 at 2 11 58 PM](https://user-images.githubusercontent.com/1776069/235522780-6c99eae4-07ae-40ff-aa8c-bd627866d91c.png)

Reply:
@powellkerry the incomplete bars should be #919191

Reply:
When I inspect, it looks like the component stylesheet is overriding the background color with the same color:

<img width="1031" alt="Screenshot 2023-05-01 at 4 31 29 PM" src="https://user-images.githubusercontent.com/872479/235535375-70ffec83-9d4d-4b88-bd2a-ad047362fa26.png">

And that is the same color as the [USWDS site](https://designsystem.digital.gov/components/step-indicator/) `#162e51`:

<img width="747" alt="Screenshot 2023-05-01 at 4 34 43 PM" src="https://user-images.githubusercontent.com/872479/235535643-0399f53e-5f6b-408c-b3c6-5d9c8558701e.png">

<img width="927" alt="Screenshot 2023-05-01 at 4 35 26 PM" src="https://user-images.githubusercontent.com/872479/235535699-b1265dcd-3be0-461b-85c1-293e9461cad2.png">

What happens if we remove all of the v3 module style overrides and just let the imported modules do the work?

Reply:
>  the incomplete bars should be #919191

@LillyBoot Is this a specific decision we have made for the VA version? Or is that color coming from somewhere else in our Sketch?

Reply:
@powellkerry I think the problem might be that we are not the latest release of USWDS. They made an update in `3.4.0` to [improve color contrast](https://github.com/uswds/uswds/pull/5048). Can you bump the `@uswds/uswds` package version and check if that fixes the incomplete bar color?

<img width="909" alt="Screenshot 2023-05-01 at 5 10 03 PM" src="https://user-images.githubusercontent.com/872479/235540329-d027c511-3eb1-4591-9b1b-c2da88a55fbc.png">

Reply:
Nice investigation work! upgrading to 3.4.0 worked.